package.json

```diff
@@ -34,6 +35,10 @@
 		"husky": "^9.1.7",
 		"lint-staged": "^16.1.2",
 		"postcss": "^8.5.6",
-		"tailwindcss": "^4.1.11"
+		"tailwindcss": "^3.4.14",
```

Comment:
why have you downgraded tailwind version!!!!
A complete migration to 4.x was done before : (

plz upgrade to 4.x latest
not downgrade to fix - that's not the way to go